pd/src/staking.rs

```diff
@@ -1,12 +1,32 @@
+use num::bigint;
+use num::rational;
+use penumbra_crypto::{Address, Value};
+use serde::{Deserialize, Serialize};
 use tendermint::{vote, PublicKey};
 
 const PENUMBRA_BECH32_VALIDATOR_PREFIX: &str = "penumbravalpub";
 /// Validator tracks the Penumbra validator's long-term consensus key (tm_pubkey), as well as their
 /// voting power.
-#[derive(Debug, Eq, PartialOrd, Ord, Clone, Copy)]
+#[derive(Deserialize, Serialize, Debug, Eq, Clone)]
 pub struct Validator {
+    /// tm_pubkey is the validator's long-term Tendermint public key, where the private component
+    /// is used to sign blocks.
     tm_pubkey: PublicKey,
+
+    /// voting_power is the validator's current voting power.
     pub voting_power: vote::Power,
+
+    /// commission_address is the validator's shielded commission address, where they receive their
+    /// portion of the staking rewards.
+    #[serde(with = "serde_with::rust::display_fromstr")]
```

Comment:
(non-blocking on this PR) Should we implement `Serialize`, `Deserialize` for `Address` so that we don't need this annotation on any struct using them that wants to use Serde?